src/lib/registration/widgets/remote_packages_table.rb

```diff
@@ -82,7 +75,11 @@ def format_items(items)
       #
       # @param item [RemotePackage]
       def columns_for_item(item)
-        [package_status(item), item.name, item.addon.name]
+        [
+          package_status(item),
+          item.name,
+          item.addon ? item.addon.name : ""
```

Comment:
We could use some new Ruby feature here...

```suggestion
          item.addon&.name.to_s
```